### ISSUE TITLE
Error about normalize_data_format

### ISSUE THREAD

**Issue by wubinbai** [titu1994/DenseNet#63]
#from keras.backend import normalize_data_format
from keras.utils.conv_utils import normalize_data_format

should replace first line of code to the second in subpixel.py
to run.
